kale/evaluate/metrics.py

```diff
@@ -45,6 +45,21 @@ def cross_entropy_logits(output, target, weights=None):
     return loss, is_correct
 
 
+def binary_cross_entropy(output, target):
```

Comment:
If `cross_entropy_logits` above is not applicable, please consider merge these two functions.